Source/Core/Core/IOS/FS/FileSystem.h

```diff
@@ -9,6 +9,11 @@
 #include <string>
 #include <vector>
 
+#ifdef _WIN32
+// TODO: Horrible hack, remove ASAP!
```

Comment:
This was mentioned on the other PR already, but could you add some more detail to this comment _why_ the hack is necessary?

Reply:
Wild guess, it's to do with the CreateDirectory macro. I'd propose renaming the function instead.

Reply:
Outside of the scope of the PR IMO.